MIP104/MIP104.md

```diff
@@ -277,6 +279,23 @@ The Stability Facilitators can at any time use the weekly governance cycle to di
 
 The Stability Facilitators can use the urgent governance cycle to propose non-standard changes to the Dai Savings Rate and the Base Rate in case of risks to the peg stability of Dai. Once the risks have subsided, the Dai Savings Rate and Base Rate must be reverted to their ordinary values, according to *3.1.1A* and *3.2.1A*.
 
+### 3.4: Core Stability Parameters Update Frequency
+
+This element and subelements define the minimum frequency of when they must be updated and implemented in the protocol. All parameters which are derived from the Core Stability Parameters must be updated at the same time, such as Native Vault Engine. Any other element which is more specific to a set of parameters or describes unique instances such as emergencies overrides this element.
+The Core Stability Parameters update is subject to three different conditions defined in the subelements 
```

Comment:
Add line break so that this is consistent with the remainder of the Scope Document.

```suggestion

The Core Stability Parameters update is subject to three different conditions defined in the subelements 
```

Reply:
Disregarded for the time being, as discussed elsewhere